lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -19,6 +20,8 @@ const removeMultiplicationByOne = require('./removeMultiplicationByOne');
 const simplifyDoubleUnaryMinus = require('./simplifyDoubleUnaryMinus');
 
 const SIMPLIFICATION_FUNCTIONS = [
+  // convert mixed numbers to improper fractions
+  convertMixedNumberToImproperFraction,
```

Comment:
Not sure where this should come in the ordering, but needs to be somewhere near the top ...